src/drivers/driver_zephyr.c

```diff
@@ -1544,11 +1612,208 @@ int wpa_drv_zep_deinit_ap(void *priv)
 		goto out;
 	}
 
+out:
+	return ret;
+}
+
+int wpa_drv_zep_sta_add(void *priv, struct hostapd_sta_add_params *params)
+{
+	struct zep_drv_if_ctx *if_ctx = priv;
+	const struct zep_wpa_supp_dev_ops *dev_ops = NULL;
```

Comment:
BTW, this `dev_ops = NULL` can be omitted because you are setting the `dev_ops` few lines below. Same comment to all similar lines in this file.